airport_data/airport_data.py

```
@@ -3,76 +3,54 @@
 import os
 
 class AirportData:
-    package_dir = os.path.dirname(__file__)
-    airports_file = os.path.join(package_dir, "airports.dat")
-    columns = ["Airport ID", "Name", "City", "Country", "IATA", "ICAO", "Latitude", "Longitude", "Altitude", "Timezone", "DST", "Tz Database Timezone", "Type", "Source"]
-    df = pd.read_csv(airports_file, names=columns, header=None)
+    columns = ["Airport ID", "Name", "City", "Country", "IATA", "ICAO", "Latitude", "Longitude", 
+               "Altitude", "Timezone", "DST", "Tz Database Timezone", "Type", "Source"]
+    
+    try:
+        df = pd.read_csv("airports_data.csv", names=columns, header=None)
+    except Exception as e:
+        df = None
```

I think we'd want to rethrow the exception here, right?